assemblyline_v4_service/common/base.py

```diff
@@ -65,10 +65,14 @@ def __init__(self, config: Optional[Dict] = None) -> None:
         self._task = None
 
         self._working_directory = None
-        self.dependencies = self._get_dependencies_info()
 
+        # Initialize interface for interacting with system safelist
+        self.api_interface = self.get_api_interface()
```

Comment:
Does the initialization of the api interface causes issues if it's never used ? Will this prevent run_service_once type of functionality to stop working?

Reply:
Yes it would